src/Image.php

```diff
@@ -767,30 +767,30 @@ public static function newFromBuffer(
 
     /**
      * Find the name of the load oepration vips will use to load a buffer, for
-     * example "VipsForeignLoadJpegBuffer". You can use this to work out what
+     * example 'VipsForeignLoadJpegBuffer'. You can use this to work out what
      * options to pass to newFromBuffer().
      *
-     * @param string $filename The formatted image to test.
+     * @param string $buffer The formatted image to test.
      *
      * @return string|null The name of the load operation, or null.
      */
     public static function findLoadBuffer(string $buffer): string
     {
-        # added in 1.0.5 of the binary module
-        if (version_compare(phpversion("vips"), "1.0.5") >= 0) {
-            $result = vips_foreign_find_load($filename);
+        // added in 1.0.5 of the binary module
+        if (function_exists('vips_foreign_find_load_buffer')) {
+            $result = vips_foreign_find_load_buffer($buffer);
```

Comment:
Travis didn't catch this because it did not execute this line. If it was executed, then this `E_NOTICE` appeared: `Notice: Undefined variable: $filename`.